modules/command-engine/core/src/main/scala/surge/internal/persistence/PersistentActor.scala

```diff
@@ -198,6 +190,11 @@ class PersistentActor[S, M, R, E](
   private val publishStateOnly: Boolean = businessLogic.kafka.eventsTopicOpt.isEmpty
 
   assert(!publishStateOnly || businessLogic.eventWriteFormattingOpt.nonEmpty, "businessLogic.eventWriteFormattingOpt may not be none when publishing events")
+
+  override def messageNameForTracedMessages: MessageNameExtractor = { case t: ProcessMessage[_] =>
+    s"ProcessMessage[${t.message.getClass.getSimpleName}]"
+  }
```

Comment:
ActorWithTracing now provides users the opportunity to provide a "pretty name" for actor messages via this partial function. If a nice name is not defined for a given message, we default .getClass.getSimpleName (which doesn't always work well ...)